_posts/2024-11-20-release.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Release 2024.47
+author: Priya
+date: 2024-11-20
+category: Jekyll
+layout: post
+---
+### Defect Fixes:
+- Resolved an issue where the math rule did not function correctly with formatted column names.
```

Comment:
Could you please change it to - "Resolved an issue that caused the math rule to malfunction when used with formatted column names."